pkg/kritis/metadata/grafeas/grafeas.go

```diff
@@ -274,3 +275,76 @@ func (c Client) fetchAttestationOccurrence(containerImage string, kind string, a
 	}
 	return occs, nil
 }
+
+func getProjectFromContainerImage(image string) string {
```

Comment:
It seems like this already exists in the containeranalysis client, no?  Can we move it to some common place?

Reply:
Ack. Yeah, we should reuse this.